src/main/java/com/fakng/fakngagrgtr/parser/uber/UberParser.java

```diff
@@ -0,0 +1,84 @@
+package com.fakng.fakngagrgtr.parser.uber;
+
+import com.fakng.fakngagrgtr.parser.ApiParser;
+import com.fakng.fakngagrgtr.parser.LocationProcessor;
+import com.fakng.fakngagrgtr.persistent.company.CompanyRepository;
+import com.fakng.fakngagrgtr.persistent.vacancy.Vacancy;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import javax.annotation.PostConstruct;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class UberParser extends ApiParser {
+
+    private static final String VACANCY_URL_FORMAT = "https://www.uber.com/global/en/careers/list/%s/";
+    private static final String CSRF_TOKEN_HEADER = "X-Csrf-Token";
```

Comment:
I think it's bad. X-Csrf-Token - standard HTTP request header key, it's not our constant. It's like expanding "Content-Type" to class constant.

Reply:
If there's a standard constant you know in Java/Spring available for this header, I'm happy to use this one
Currently, I can find one only in the Spring Security package. Any alternatives?